etc/httpd/global.conf

```diff
@@ -242,6 +242,9 @@ RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?download/?$ /$1firefox/new/ [L,R=301]
 # bug 957283
 RewriteRule ^/(pt-BR)/firefox/speed(/?)$ /b/$1/firefox/speed$2 [PT]
 
+# bug 1146982
+RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?firefox/send-to-device-post(/?)$ /b/$1firefox/send-to-device-post$2 [PT]
```

Comment:
This is just because we typically do this. If we don't capture the trailing slash then going to the URL w/o the slash will 404, but if it goes to bedrock it'll 302. If we proxy both urls to the url with the slash on the backend then I think there are some odd redirect behaviors.